WINDOWS_LINUX_SUBSYSTEM2.md

```diff
@@ -0,0 +1,257 @@
+# Setup your machine!
+
+Welcome to this setup where we are going to install the tools required to make your machine a true developer environment :muscle:
+
+
+
+## Prerequisites
+
+Before we start it is important you meet the following prerequisites to ensure the smooth running of the setup.
+Take your time to read through everything and do not hesitate to ask for help if you feel stuck.
+Ready? Let's go :sunglasses:
+
+
+
+### Latest version of Windows
+
+For this setup, you need to be running on the latest version of Windows.
+
+This means that you need to be on Windows 10 with all the latest updates installed.
+
+You can check your software version by clicking on **Start>Settings>System>About**, look where it says *edition*. If you see something that starts with "Windows 10..." you're good to go :muscle:.
+
+Not the case? Don't panic :scream: You can always install windows 10 from [microsoft]( https://www.microsoft.com/en-gb/windows/get-windows-10) and click on *Check for Updates* then follow the instructions on screen. Come back to this setup when Windows 10 is installed.
+
+Once you are sure that you're running Windows 10, you will need to check that your machine is running with all the latest updates. For this click on **Start>Settings>Updates & Security>Windows Update** then click on **Check updates**. If you have updates available please install them and repeat until it says that you are up to date :star:.
+
+
+
+### The insider program
+
+As a soon-to-be-developer, you always want to get the beta version of everything :nerd_face:.
+
+This also applies to Windows, so we are going to sign up to their beta program, the **Insider program** .
+
+Go to **Start>Settings>Updates & Security>Insider Program**
+
+Click on **Get Started**. It will ask you to **Link an account**; follow the instructions on screen. You will be prompted (code-speak for being asked) to choose your insider parameters, choose the second setting, the **fast** one :zap:. Then confirm and restart your computer when you are prompted to do so.
```

Comment:
maybe we can simplify and just replace `prompted` by `asked` as this setup is a lot longer than the others and might be scary for newbies 🙈